pkg/apimachinery/discoverysummarizer/server/main.go

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Moved to `cmd/kubernetes-discover`.
Chose `kubernetes-discover` over `kube-discover` as per comment here: https://github.com/kubernetes/kubernetes/pull/23509#issuecomment-203083668